pan/legacy.pan

```diff
@@ -1,3 +1,12 @@
 declaration template pan/legacy;
 
 type legacy_binary_affirmation_string = string with match(SELF, "^(yes|no)$");
+
+function is_yes_no_true_false = {
+    v = ARGV[0];
+    if (is_boolean(v)) return(true);
+    if (is_string(v) && match(v, "^(yes|no)$")) return(true);
```

Comment:
As for me, the first goal is to discourage people to use this inappropriate type... As for migrating the existing use, I think we need a minimum of evaluation about the impact (basically how many components will be affected by a non backward compatible schema)?